packages/moonstone/Input/Input.module.less

```diff
@@ -168,7 +168,7 @@
 
 			.input,
 			.decoratorIcon {
-				color: @moon-input-text-disabled-color;
+				color: @moon-input-text-disabled-focus-color;
```

Comment:
@Djspaceg Should we move this into a `.focus` section?  Also, do we need to update the `.focus()` to also or instead us `focus-within`?

Reply:
This new variable does appear to not be in the ideal place (its name suggests it should be inside a disabled block and a focus block, while it's currently just inside a disabled block). It also seems to delete any reference to `@moon-input-text-disabled-color` which I don't think is the intent.